django_cloud_deploy/cli/prompt_manager.py

```diff
@@ -0,0 +1,673 @@
+import abc
```

Comment:
Do you have the usage of `prompt_manager` done? Could I see a usage example?

Reply:
Will add to next commit after fixes.